scripts/imgtool/image.py

```diff
@@ -255,9 +258,10 @@ def ecies_p256_hkdf(self, enckey, plainkey):
             format=PublicFormat.UncompressedPoint)
         return cipherkey, ciphermac, pubk
 
-    def create(self, key, enckey, dependencies=None):
+    def create(self, key, enckey, dependencies=None, custom_protected_tlv=None):
         self.enckey = enckey
 
+        dependencies_num = 0
```

Comment:
Why was this added? To me it looks like it is only initialized and use if `dependencies is not None` which should be OK.